src/buffer/out/textBuffer.hpp

```diff
@@ -64,6 +64,41 @@ namespace Microsoft::Console::Render
     class Renderer;
 }
 
+enum class MarkCategory : size_t

```

Comment:
FYI you can use `uint8_t` here or just remove the explicit underlying type entirely.